src/main/java/com/aleksey5678/dividing/Positions.java

```diff
@@ -0,0 +1,48 @@
+package com.aleksey5678.dividing;
+
+public class Positions {
+    private static final int POSITION_OF_DIVISOR = 0;
+    private static final int POSITION_OF_LINE_SPLITTER = 1;
+    private static final int POSITION_OF_QUOTIENT = 2;
+    private final int positionOfDivisor;
```

Comment:
please separate static and non-static members of class with empty line. This will improve readability

Reply:
Done